docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_marquee.htm

```diff
@@ -13,7 +13,7 @@ <h1>Marquee</h1>
 It makes &lt;div&gt; element (has some child nodes such as text and img) move horizontally like legacy &lt;marquee&gt; tag.</p>
 
 <video autoplay muted loop controls>
-	<source src="../../images/Marquee.ogv">
+	<source src="https://developer.tizen.org/sites/default/files/marquee.ogv">
```

Comment:
Like src="https://developer.tizen.org/sites/default/files/documentation/marqee.ogv"
Would you add the  /documentation/ , please?

Reply:
And we have to change the video format ogv->mp4.
Because ogv format don't play in the IE.

Reply:
I added "/documentation" in file path, and change video extension, but I change video files in separate PR.

Reply:
Done.